### PR TITLE
Hello World in MojiScript

### LINKED ISSUE #550: Add Hello World in MojiScript
Just found out about this over on [Dev.to](https://dev.to/joelnet/comment/66k3). Looks cool! Let's get their language represented!

Repo: https://github.com/joelnet/MojiScript/

### PR DESCRIPTION
**Pull request by Piglacquer**
Congrats on taking the first step to contributing to the [Sample Programs in Every Language][1] repository!

For simplicity, please make sure that your pull request includes one and only one sample program and language.

## Please Complete the Following

- [ ] I would **NOT** like credit for my contribution in the article (credit will be included otherwise)
- [ ] I plan to write the article for this code snippet
- [x] I named the pull request using `<Sample Program> in <Language>` format
- [x] I created/updated the language README
  - [x] I added the sample program name to the README
  - [ ] I added fun facts (i.e. debut, developer, typing, etc.)
  - [x] I added reference link(s) to the README
  - [x] I added solution citations when necessary (see [plagiarism][2])
- [ ] I fixed #your-issue-number-here

## Notes

First PR to a publicly maintained repo  🎉
Fixes #550 

[1]: https://therenegadecoder.com/code/sample-programs-in-every-language/
[2]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/CONTRIBUTING.md#plagiarism

### CODE REVIEW

**Comment by jrg94** [comment]
Hey @Piglacquer! You would need to create a new issue--not rename this pull request. Hope that makes more sense!

**Comment by Piglacquer** [comment]
Added issue #557 . Thanks for your patience!

**Comment by Noah11012** [comment]
Everything looks good.

File: archive/m/mojiscript/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Sample Programs in MojiScript
+
+- hello-world.mjs
```
Comment (jrg94) [inline archive/m/mojiscript/README.md]: I know this got merged, but the README is inconsistent with the rest of the repo. Can we revisit this?

File: archive/m/mojiscript/hello-world.mjs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import log from 'mojiscript/console/log'
+import pipe from 'mojiscript/core/pipe'
+import run from 'mojiscript/core/run'
+
+const state = 'Hello World'
```
Comment (jrg94) [inline archive/m/mojiscript/hello-world.mjs]: Also, for consistency, we should probably update this string to "Hello, World!"

**Pull request merged**